locales/es.js

```diff
@@ -67,6 +67,7 @@ export default {
     cartocode: 'Cartocode',
     noResult: 'No hay resultados',
     hideCategory: 'Ocultar esta categoría',
+    clearAllCategories: 'Ocultar todo los categoría',
```

Comment:
Not really Spanish. Just omiit Spanish if you cannot wrtie it. I will do.
```suggestion
    clearAllCategories: 'Ocultar todas las categorías',
```